handlers.js

```diff
@@ -239,10 +241,35 @@ async function checkFilePropagationServerReachableHandler() {
   }
 }
 
-async function runTests(){
-  await checkChiaConfigIpHostHandler();
-  await checkFilePropagationServerReachableHandler();
-  return true;
+function saveFileListToJson(fileList, directory, storeId) {
+  try {
+    const jsonFilePath = path.join(directory, `/manifests/${storeId}.json`);
+    const jsonData = {
+      directory,
+      storeId,
+      files: fileList.map(file => ({
+        key: file.key,
+        value: file.value
+      }))
+    };
+
+    fs.writeJsonSync(jsonFilePath, jsonData, { spaces: 2 });
+    console.log(`File list saved as JSON in ${jsonFilePath}`);
+    return jsonData; // Returning jsonData instead of fs's writeJsonSync return value
+  } catch (error) {
+    console.error('Error saving file list:', error);
+  }
+}
+
+async function runTests() {
```

Comment:
make sure you pulled the latest main down becuase i actually tweak the runTests() logic a little bit before i deployed it. (example i renamed it "runConnectionCheckHandler" )